test/code/plugins/filter_changetracking_test.rb

```diff
@@ -26,6 +27,9 @@ def teardown
       if File.exist?(@fileinventoryHashPath1)
          File.delete(@fileinventoryHashPath1)
       end
+      if File.exist?(@fileinventorytimestampHashPath)
```

Comment:
can you follow same style for parameter names for consistency? Inventory should be uppercase.